app/src/main/java/mozilla/lockbox/store/RouteStore.kt

```diff
@@ -19,8 +23,15 @@ import mozilla.lockbox.support.asOptional
 @ExperimentalCoroutinesApi
 class RouteStore(
     dispatcher: Dispatcher = Dispatcher.shared,
-    dataStore: DataStore = DataStore.shared
+    dataStore: DataStore = DataStore.shared,
+    fingerprintStore: FingerprintStore = FingerprintStore.shared
 ) {
+    internal val compositeDisposable = CompositeDisposable()
+
+    private val onboardingState: ReplaySubject<Boolean> = ReplaySubject.createWithSize(1)
```

Comment:
a few things:

let's put all the properties _after_ the `companion object` declaration so that we don't have the lonely `routes` object down below, and unify our `Observable` declarations so that they all look like:

```kotlin
val onboardingState: Observable<Boolean> = ReplaySubject.createWithSize(1)
```

Reply:
👏